mithril-common/src/crypto_helper/types/protocol_key.rs

```diff
@@ -285,4 +287,25 @@ mod test {
 
         assert_eq!(expected_key, read_key);
     }
+
+    #[test]
+    fn can_read_a_verification_key_from_file_with_trailing_whitespaces() {
```

Comment:
Since the fix doesn't involve files in any-ways should the test use files ?

Reply:
The problem occurred when creating a key file with a copy paste, this is why it's tested here.